flow/designs/asap7/megaboom/pdn.tcl

```diff
@@ -0,0 +1,54 @@
+####################################
+# global connections
```

Comment:
I have zero experience w PDN :-)

Reply:
That will change!
Your design routeability ... IR ... will get you going.